025-control-plain-listener.md

```diff
@@ -0,0 +1,80 @@
+# Control Plane Listener
+
+This proposal suggests to use separate Control Plane listener and the plan how it should be introduced.
+
+## Current situation
+
+Strimzi currently has one internal listener.
+It is used by the Kafka brokers for data replication but also for coordination between the controller and the other brokers.
+And in addition to that, it is also used by the different Strimzi components:
+* Operators
+* Kafka Exporter
+* Cruise Control
+
+## Motivation
+
+Kafka already for some time supports using separate listeners for data replication and coordination.
```

Comment:
Can we link to the KIP which added the feature?

Reply:
Is it [KIP-103](https://cwiki.apache.org/confluence/display/KAFKA/KIP-103%3A+Separation+of+Internal+and+External+traffic) and/or [KIP-291](https://cwiki.apache.org/confluence/display/KAFKA/KIP-291%3A+Separating+controller+connections+and+requests+from+the+data+plane)?

Reply:
It's 291

Reply:
Yeah, its 291.